lbrynet/tests/unit/core/test_Wallet.py

```diff
@@ -195,3 +208,59 @@ def update_balance():
             'test', "f43dc06256a69988bdbea09a58c80493ba15dcfa", 4))
         self.assertFailure(d, InsufficientFundsError)
         return d
+
```

Comment:
These tests look like they belong in lbryum since no lbrynet objects are used here 
I think moving this to new file under lbryum/tests like test_encryption.py would be good.

Reply:
@kaykurokawa I have refactored the tests to make use of common setUp and tearDown code. I think there's a benefit of having some of these tests here to just to make sure that they work properly when called from the daemon. We can add more tests to the lbryum suites.

I think we should create additional issues to address creating improved accuracy tests in the lbrynet test suites as well as the lbryum test suites.